test/test_population.py

```diff
@@ -0,0 +1,69 @@
+import copy
+import gp
+import pytest
+import numpy as np
+
+
+def test_assert_mutation_rate(rng_seed, genome_params):
+    with pytest.raises(ValueError):
+        gp.Population(5, -0.1, rng_seed, genome_params)
+
+    with pytest.raises(ValueError):
+        gp.Population(5, 1.1, rng_seed, genome_params)
+
+
+def test_init_random_parent_population(population_params, genome_params):
+    pop = gp.Population(**population_params, genome_params=genome_params)
+    assert len(pop.parents) == population_params["n_parents"]
+
+
+def test_champion(population_params, genome_params):
+    pop = gp.Population(**population_params, genome_params=genome_params)
+    for i, parent in enumerate(pop.parents):
+        parent.fitness = float(i)
```

Comment:
I am assuming that fitness value is supposed to be a float. Strictly speaking, it can be anything of course, but in general, it should be a `float`, not an `int`, shouldn't it?

Reply:
puh, i guess in most cases it would be a float, but it don't think we need to enforce that. anything that can be compared/ordered could serve as a fitness, couldn't it?

Reply:
Okay, changed to `int`.